src/pages/api/sanity/lessons/create.ts

```diff
@@ -36,6 +36,7 @@ type SanityLesson = {
   title: string
   description?: string
   repoUrl?: string
+  softwareLibraries: SanitySoftwareLibrary[]
```

Comment:
Chose to have this as an array since that reflects the data type in Sanity. We may also want to expand on lesson tagging in the upload UI in the future, potentially giving lessons multiple tags

Reply:
I expect we will eventually add support for multiple tags in the Course Builder UI. That would involve pulling in, configuring, and styling something like `react-select`. Seems best to punt on that until we are in more of a v2 frame of mind.